MIGRATION_GUIDE.md

```diff
@@ -20,6 +20,14 @@ Its purpose was also to divide the resources by their specific purpose rather th
 
 Renamed field `provisioner_role` to `run_as_role` to align with Snowflake docs.
```

Comment:
Let's add, that config should be renamed and the state would be migrated automatically

Reply:
Done.